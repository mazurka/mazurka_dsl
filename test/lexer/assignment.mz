
Users <- 123
Other  <-      test