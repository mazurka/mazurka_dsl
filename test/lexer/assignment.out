{variable,2,'Users'}.
{assign,2}.
{integer,2,123}.
{variable,3,'Other'}.
{assign,3}.
{atom,3,test}.